pkgs/tools/misc/fontforge/default.nix

```diff
@@ -10,6 +10,8 @@
 stdenv.mkDerivation rec {
   name = "fontforge-${version}";
   version = "20160404";
+  versionModtime = "1459728000"; # unix timestamp of ${version}
```

Comment:
Can we do `date -d $version +%s` or something here? Ditto for the one below? If it matters that they're the same, that is.